prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -567,8 +574,9 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
             return
         }
 
-        // Output the data for review.
-        if (!silent) {
+        // Output the data for review specified and/or the table is small enough.
+        val isLargeTable = inputData.size > 50 || inputData[0].keys.size > 7
+        if (!silent && (showTable || (!showTable && !isLargeTable))) {
```

Comment:
if (!silent && (showTable || !isLargeTable)) {
showTable's existence shortcuts to "true"... otherwise you only get a true if !isLargeTable.

Reply:
Well, not exactly.  The table will be printed:
1. only when silent is set to false AND
2. 
  a. show-table is true OR
  b. show-table is false AND the table is small (not large)

Am I missing something?

Reply:
Either way.  You're good.  In fact, explicit is better, according to Guido.
In fact,
if (!silent && (!isLargeTable || showTable)) {
woulda been clearer.